interface/modules/custom_modules/oe-module-claimrev-connect/public/x12Tracker.php

```diff
@@ -0,0 +1,126 @@
+<?php
+    namespace OpenEMR\Modules\ClaimRevConnector;
+    $tab="x12";
+    require_once "../../../../globals.php";
+
+    use OpenEMR\Modules\ClaimRevConnector\Bootstrap;
+    use OpenEMR\Modules\ClaimRevConnector\ClaimRevApi;
+
+    class X12TrackerPage
```

Comment:
classes should be in src directory with other classes

Reply:
done